modules/twinklexfd.js

```diff
@@ -1242,20 +1243,22 @@ Twinkle.xfd.callbacks = {
 		// This is a closure for the callback from the above API request, which gets the target of the redirect
 		findTargetCallback: function(callback) {
 			return function(apiobj) {
-				var xmlDoc = apiobj.responseXML;
-				var target = $(xmlDoc).find('redirects r').first().attr('to');
+				var $xmlDoc = $(apiobj.responseXML);
```

Comment:
Just noting that elsewhere in `xfd` we don't use the jquery naming convention; this is the only one that actually uses it for more than convenience, so it's reasonable to do here.